pkg/activator/handler/handler_test.go

```diff
@@ -123,7 +123,7 @@ func TestActivationHandler(t *testing.T) {
 
 			ctx, cancel, _ := rtesting.SetupFakeContextWithCancel(t)
 			defer cancel()
-			handler := New(ctx, test.throttler, rt, false /*usePassthroughLb*/, logging.FromContext(ctx))
+			handler := New(ctx, test.throttler, rt, false /*usePassthroughLb*/, logging.FromContext(ctx), false /* TLS */)
```

Comment:
Could we add a test where this is true?

Reply:
I think it is not possible to add the HTTPs proxy in this test code. `TestNewHeaderPruningProxyHTTPS` covered the part instead.